src/main/java/org/opentripplanner/netex/NetexBundle.java

```diff
@@ -171,13 +171,14 @@ private void loadFilesThenMapToOtpTransitModel(
 
     // map current NeTEx objects into the OTP Transit Model
     mapper.mapNetexToOtp(index.readOnlyView());
+    issueStore.stopProcessingSource();
   }
 
   /** Load a single entry and store it in the index for later */
   private void loadSingeFileEntry(String fileDescription, DataSource entry) {
     try {
       LOG.info("reading entity {}: {}", fileDescription, entry.name());
-
+      issueStore.startProcessingSource(entry.name());
       PublicationDeliveryStructure doc = xmlParser.parseXmlDoc(entry.asInputStream());
       NetexDocumentParser.parseAndPopulateIndex(index, doc, ignoreFareFrame);
     } catch (JAXBException e) {
```

Comment:
Why not ?
```
} finally {
  issueStore.stopProcessingSource();
}
```
The validation that happens in the Validator will report on the wrong file, so it should not have a source set...